app/templates/components/goodcity/orders-package-block.hbs

```diff
@@ -57,18 +57,4 @@
         {{action-drawer isOpen=drawerOpened actionList=actionList small=small}}
     </div>
   {{/unless}}
-
```

Comment:
@swatijadhav Can you confirm that removing this is okay ? Since we're using the new service entry point you created

Reply:
@patrixr this code seems to be for dispatch form. In the designation-confirmation changes PR https://github.com/crossroads/stock.goodcity/pull/898 we have already removed code changes for designation overlay here
https://github.com/crossroads/stock.goodcity/pull/898/files#diff-4365678f4a18a6be110bcf81c2f18a4d5b7a8aa3d1f6447d597e0c6b2d4cc911

Can you please have a look?

Reply:
ooooh thanks : D